src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -806,8 +806,50 @@ parseMPToken(Json::Value const& mptJson, Json::Value& jvResult)
     }
 }
 
+static std::optional<uint256>
+parsePermissionedDomains(Json::Value const& pd, Json::Value& jvResult)
+{
+    if (pd.isString())
+    {
+        Json::Value result;
+        auto const index = parseIndex(pd, result);
+        if (!index)
+            jvResult[jss::error] = "malformedObjectId";
+        return index;
+    }
+
+    if (!pd.isObject())
+    {
+        jvResult[jss::error] = "malformedObject";
+        return std::nullopt;
+    }
+
+    if (!pd.isMember(jss::account) || !pd[jss::account].isString())
+    {
+        jvResult[jss::error] = "malformedAccount";
+        return std::nullopt;
+    }
+
+    if (!pd.isMember(jss::seq) ||
+        (pd[jss::seq].isInt() && pd[jss::seq].asInt() < 0) ||
+        (!pd[jss::seq].isInt() && !pd[jss::seq].isUInt()))
```

Comment:
Can this be replaced with `pd[jss::seq].isConvertibleTo(Json::uintValue)`?

Reply:
I don't think it should. These are strict checks vs type conversion